glhe/output_processor/output_processor.py

```diff
@@ -24,30 +22,40 @@ def collect_output(self, data_dict: dict) -> None:
 
         :param data_dict: dictionary of data to be logged
         """
-
-        df_temp = pd.DataFrame(data_dict, index=[self.idx_count])
-        self.df = pd.concat([self.df, df_temp], axis=0, sort=True)
-        self.idx_count += 1
+        self.output_data.append(data_dict)
```

Comment:
The output processor could use some love.  It needs a round of efficiency upgrading, but it does seem to be working happily without pandas, which is the right step.